File: app/src/main/java/ladsoft/com/popularmoviesapp/api/parser/MovieVideosRequestResult.java
package ladsoft.com.popularmoviesapp.api.parser;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import ladsoft.com.popularmoviesapp.model.MovieVideo;

public class MovieVideosRequestResult {

    @Expose @SerializedName("id") private long id;
    @Expose @SerializedName("results") private List<MovieVideo> results;

    public long getId() { return id; }
    public List<MovieVideo> getResults() { return results; }
}

File: app/src/main/java/ladsoft/com/popularmoviesapp/api/parser/MovieVideosRequestResult.kt
package ladsoft.com.popularmoviesapp.api.parser


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import ladsoft.com.popularmoviesapp.model.MovieVideo

data class MovieVideosRequestResult(@Expose @SerializedName("id") val id: Long = 0.toLong(),
    @Expose @SerializedName("results") val results: List<MovieVideo> = emptyList())
